v23.1/show-backup.md

```diff
@@ -77,6 +77,7 @@ Field | Description
 `rows` | Number of rows in tables that are part of the backup.
 `create_statement` | The `CREATE` statement used to create [table(s)](create-table.html), [view(s)](create-view.html), or [sequence(s)](create-sequence.html) that are stored within the backup. This displays when `SHOW BACKUP SCHEMAS` is used. Note that tables with references to [foreign keys](foreign-key.html) will only display foreign key constraints if the table to which the constraint relates to is also included in the backup.
 `is_full_cluster` |  Whether the backup is of a full cluster or not.
+`regions` | The [`ALTER DATABASE`](alter-database.html) statement(s) used to configure the database with [multi-region zone configurations](multiregion-overview.html#database-regions), if they exist. If the database is single-region, `NULL` will show. 
```

Comment:
s/single-region/apart of single region cluster configuration/r

In an MR cluster configuration, all Databases will need to be configured with an `ALTER DATABASE` cmd. See these [instructions](https://www.cockroachlabs.com/docs/v23.1/multiregion-overview#overview). In an MR cluster configuration, you could still have a database constrained to a single region.